.github/workflows/golangci-lint.yml

```diff
@@ -15,4 +18,6 @@ jobs:
       - name: golangci-lint
         uses: golangci/golangci-lint-action@v2
         with:
-          version: v1.42
+          version: v1.43
+          # Optional: show only new issues if it's a pull request. The default value is `false`.
+          only-new-issues: true
```

Comment:
The linter was failing because it was requiring all the headers to be fixed (year 2022 instead of 2021). 

This should not be a requirement except for new/modified files, so the action now just checks new changes, similarly to what the local linter does.